python/lsst/analysis/tools/analysisPlots/skyObject.py

```diff
@@ -61,16 +61,27 @@ def setDefaults(self):
         self.prep.selectors.skyObjectSelector = SkyObjectSelector()
 
         self.process.buildActions.hist_psf_flux = LoadVector(vectorKey="{band}_psfFlux")
-        self.process.buildActions.hist_09_flux = LoadVector(vectorKey="{band}_ap09Flux")
+        self.process.buildActions.hist_ap09_flux = LoadVector(vectorKey="{band}_ap09Flux")
+        self.process.buildActions.hist_gaap1p0_flux = LoadVector(vectorKey="{band}_gaap1p0Flux")
         self.process.buildActions.hist_psf_sn = SNCalculator(fluxType="{band}_psfFlux")
-        self.process.buildActions.hist_09_sn = SNCalculator(fluxType="{band}_ap09Flux")
+        self.process.buildActions.hist_ap09_sn = SNCalculator(fluxType="{band}_ap09Flux")
+        self.process.buildActions.hist_gaap1p0_sn = SNCalculator(fluxType="{band}_gaap1p0Flux")
```

Comment:
According to my `black ` formatter, these new lines should be wrapped around. I'd recommend wrapping all of them around for consistency using commas, i.e.:
```
        self.process.buildActions.hist_psf_flux = LoadVector(
            vectorKey="{band}_psfFlux",
        )
        self.process.buildActions.hist_ap09_flux = LoadVector(
            vectorKey="{band}_ap09Flux",
        )
        self.process.buildActions.hist_gaap1p0_flux = LoadVector(
            vectorKey="{band}_gaap1p0Flux"
        )
        self.process.buildActions.hist_psf_sn = SNCalculator(
            fluxType="{band}_psfFlux",
        )
        self.process.buildActions.hist_ap09_sn = SNCalculator(
            fluxType="{band}_ap09Flux",
        )
        self.process.buildActions.hist_gaap1p0_sn = SNCalculator(
            fluxType="{band}_gaap1p0Flux",
        )
```